crt_portal/cts_forms/templates/forms/complaint_view/actions/bulk_assign.html

```diff
@@ -0,0 +1,64 @@
+{% extends "forms/complaint_view/show/card.html" %}
+
+{% load static %}
+
+{% block title %}Bulk assign{% endblock %}
+
+{% block extra_classes %}crt-actions-card{% endblock %}
+
+{% block icon %}
+  <img src="{% static "img/ic_personcheck-circle.svg" %}" alt="icon" class="icon" />
+{% endblock %}
+
+{% block card_content %}
+  {% if ids_count %}
+    <form class="usa-form" method="post" novalidate>
+      {% csrf_token %}
+
+      <input type="hidden" value="{{ return_url_args }}" name="next" id="next" />
+      <input type="hidden" value="{{ ids }}" name="ids" id="ids" />
+      <input type="hidden" value="{{ selected_all }}" name="all" id="all" />
```

Comment:
Do we need this hidden input for `all` if the intention is to pass it thru to the `POST` with the "apply to all reports button"?

As-is, this results in applying changes to **all** reports when hitting **either** submission button on the form.

Reply:
We need to preserve the `selected_all` in case of form error. That way, if the POST should fail, the user will still have the option to "select all". 

And, you are right. I was re-using the `all` name when I didn't have to; the "confirm yes" button now has the `confirm_all` name attribute. Good catch! I'll also add tests for this case ("select all but only confirm 15").